Core/include/Acts/Propagator/EigenStepper.hpp

```diff
@@ -83,7 +84,7 @@ class EigenStepper {
     explicit State(const GeometryContext& gctx,
                    MagneticFieldProvider::Cache fieldCacheIn,
                    const BoundTrackParameters& par,
-                   double ssize = std::numeric_limits<double>::max())
+                   double ssize = 10 * Acts::UnitConstants::m)
```

Comment:
Could make this a central static default at the `EigenStepper` class level.

Is / should this be synced to the `StepperOptions` max step size?

Reply:
I think you are right, this can be replaced by the options. Let me try that